### PR TITLE
New download

### PR DESCRIPTION
**Pull request by Vital-jan**
compare filesize, download, unzip and rename files.

### CODE REVIEW

File: ratu/services/main.py [hunk 1]
```diff
@@ -1,32 +1,132 @@
 import codecs
 from collections import defaultdict
 from django.apps import apps
+import json
 import io
+import os
+from os import path
```
Comment (criticus) [inline ratu/services/main.py:7]: why do you import `os` twice?

File: script.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from ratu.services.rfop_service import RfopConverter
```
Comment (criticus) [inline script.py:1]: what is script.py for?

File: ratu/services/main.py [hunk 3]
```diff
@@ -1,32 +1,132 @@
...
 import requests
 import sys
+import time
```
Comment (criticus) [inline ratu/services/main.py:10]: where do you use `time`?

File: ratu/services/main.py (hunk 1 above)
Comment (Vital-jan) [reply ratu/services/main.py:7]: fixed

File: script.py (hunk 2 above)
Comment (Vital-jan) [reply script.py:1]: for one command run application with django_extention (your advice)

File: ratu/services/main.py (hunk 3 above)
Comment (Vital-jan) [reply ratu/services/main.py:10]: already unused. fixed.

File: script.py (hunk 2 above)
Comment (criticus) [reply script.py:1]: okay please add instructions to wiki how to run this script

Comment (Vital-jan) [reply script.py:1]: Ok. I`ll write this at wiki: python3 manage.py runscript script.py

Comment (Vital-jan) [reply script.py:1]: https://github.com/3v-workspace/Data_convertor/wiki/Django-Runscript

**Pull request merged**